src/mbgl/map/map.cpp

```diff
@@ -166,14 +153,12 @@ Map::Impl::Impl(Map& map_,
           }
       }) {
     style = std::make_unique<Style>(scheduler, fileSource, pixelRatio);
```

Comment:
Not part of this pr. But why aren't we initialising this in the member initialisation list?